doc.go

```diff
@@ -9,6 +9,8 @@ var ErrNotFound = errors.New("not found")
 // Geche interface is a common interface for all cache implementations.
 type Geche[K comparable, V any] interface {
 	Set(K, V)
+	// SetIfPresent sets the kv only if the key was already present, and returns whether the insertion was performed
```

Comment:
minor: inconsistent pub iface methods comments. It's not a big deal, but still looks a bit sloppily imho

Reply:
Will add comments on all methods for consistency in the next PR.
I want to add new function to Readme and also add `SetIfNotExists` for the symmetry's sake  if I have time.